easybuild/easyconfigs/s/Sambamba/Sambamba-0.7.1.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PackedBinary'
+
+name = 'Sambamba'
+version = '0.7.1'
+local_distbin = '%(namelower)s-%(version)s-linux-static'
+
+homepage = 'https://lomereiter.github.io/sambamba/'
+description = """Sambamba is a high performance modern robust and fast tool
+ (and library), written in the D programming language, for working with SAM
+ and BAM files. Current functionality is an important subset of samtools
+ functionality, including view, index, sort, markdup, and depth."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/biod/%(namelower)s/releases/download/v%(version)s']
+sources = [{'download_filename': '%s.gz' % local_distbin, 'filename': '%(namelower)s-%(version)s.gz'}]
```

Comment:
In this case it is probably better to keep the upstream file name intact. It contains all the info, name and version.
Keeping the upstream name makes it easier to identify what was downloaded by just doing "ls" in the source dir.

Reply:
Fixed :+1: